general_itests/steps/paasta_execute_docker_command.py

```diff
@@ -38,6 +38,7 @@ def create_docker_container(context, task_id):
         context.docker_client.remove_container(container_name, force=True)
     except APIError:
         pass
+    context.docker_client.pull('ubuntu:trusty')
```

Comment:
hm, thats fair, I could change the feature so that the image name is parameterised.